### PR TITLE
Version 23.10.0

### PR DESCRIPTION
**Pull request by SC426**
Using conda-forge recipe

### CODE REVIEW

**Review by Jrice1317 — COMMENTED** [review]
Hello! Are we purposely ignoring the linter for this recipe?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,60 +1,71 @@
...
-    sha256: b000dde0ac8641c92f7342362173968505d4e85845be37655b5b9221cb65f3a8  # [win]
-    folder: constructor  # [win]
+    sha256: 787ffd85e9414bdf70fe531f01eab3987a040e3f6a6ac3a01409f4d332f7de9e  # [win]
+    folder: constructor_src  # [win]
 
 build:
-  number: 1
+  number: 0
+  ignore_run_exports:
+    - '*'
```
Comment (Jrice1317) [inline recipe/meta.yaml:30]: Should the patches be here or is this recipe different from most? @SC426 @marcoesters

Comment (Jrice1317) [reply recipe/meta.yaml:30]: Also, why do we have two `build` sections and are missing a `host` section?

File: abs.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+build_parameters:
```
Comment (marcoesters) [inline abs.yaml:1]: Please remove the file

File: recipe/bld.bat [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
 
-MKDIR "%PREFIX%\standalone_conda"
-MOVE dist\conda.exe "%PREFIX%\standalone_conda\conda.exe"
+goto :EOF
 
-RD /s /q "%PREFIX%\lib"
+:error
+set "exitcode=%errorlevel%"
+echo Failed with error #%exitcode%.
+exit /b %exitcode%
```
Comment (marcoesters) [inline recipe/bld.bat:35]: Please add line breaks at the end of files. This is just causing unnecessary diffs compared to conda-forge

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,60 +1,71 @@
...
 
 requirements:
   build:
-    - python {{ python_version }}
-    - pyinstaller 5.6.2
-    - conda {{ version }}
-    - conda-package-handling >=1.7.2
-    - patch  # [not win]
-    - m2-patch  # [win]
+    - pyinstaller
```
Comment (marcoesters) [inline recipe/meta.yaml:34]: This is missing `patch` and `m2-patch` - see the linter output

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,60 +1,71 @@
...
-  license_file: LICENSE
+  home: https://github.com/conda/conda-standalone
   summary: Entry point and dependency collection for PyInstaller-based standalone conda.
-  description: Standalone version of conda to bootstrap conda for Miniconda and Anaconda Distribution.
+  description: A standalone conda executable built with PyInstaller.
+  license: LicenseRef-CondaStandalone
+  license_file:
+    - LICENSE
+    - 3rd-party-licenses.json
+  license_family: BSD
```
Comment (marcoesters) [inline recipe/meta.yaml:61]: This is incorrect. There is no license family because there are multiple licenses. Use `skip-lint` to skip that check

**Review by marcoesters — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml:30]: I don't think the two build sections are the same, but I could be wrong. Do we want to try to figure out what a 'host' and 'run' section would look like?

Comment (marcoesters) [reply recipe/meta.yaml:30]: 1. The patches do not belong into the build section, they need to go with each source.
2. There is only one `build` section. `requirements/build` is a different field.
3. Why do we need a `host` or `run` section?

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,60 +1,75 @@
...
 requirements:
   build:
-    - python {{ python_version }}
-    - pyinstaller 5.6.2
-    - conda {{ version }}
-    - conda-package-handling >=1.7.2
     - patch  # [not win]
     - m2-patch  # [win]
+    - pyinstaller
+    - python ={{ python_version }}
```
Comment (Jrice1317) [inline recipe/meta.yaml:37]:
```suggestion
    - python {{ python_version }}
```

Comment (Jrice1317) [reply recipe/meta.yaml:37]: @SC426 @marcoesters Is the `=` a new way of pinning?

Comment (marcoesters) [reply recipe/meta.yaml:37]: >  Is the = a new way of pinning?

Not new, just an alternative. Rule of thumb: as long as it's consistent, I wouldn't bother

**Pull request merged**